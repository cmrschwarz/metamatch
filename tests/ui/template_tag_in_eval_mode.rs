use metamatch::eval;

fn main() {
    let res = eval! {
        [<eval>]
        42
        [</eval>]
    };

    assert_eq!(res, 10);
}
