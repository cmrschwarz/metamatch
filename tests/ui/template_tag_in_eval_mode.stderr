error: template tags are not supported in eval mode
 --> tests/ui/template_tag_in_eval_mode.rs:5:9
  |
5 |         [<eval>]
  |         ^^^^^^^^
